Makefile.am

```diff
@@ -0,0 +1,30 @@
+## This is a automake file, part of the GPTL package.
+# Copyright 2018, see the COPYRIGHT file for more information.
```

Comment:
@jmrosinski do we have a copyright notice to put in a COPYRIGHT file?

Reply:
Yes there is a file COPYING in the main directory. You can either rename it, and/or feel free to modify and let me have a look.